### DISCUSSION TITLE
How to send custom argument alongside default arguments in slide item?

### DISCUSSION THREAD

**Discussion by meo1** [quasarframework/quasar#13933] [General - Components / Directives / etc] [answered]
For example:

<q-slide-item @slide="onSlide(element.id)"> 

function onSlide ({ side, ratio, isReset }, elementID) {
console.log("ratio: " + ratio + "element: " + elementID)
} 

![image](https://user-images.githubusercontent.com/35503088/178496873-333d381d-6352-4d92-ac9e-b3a51b8684f9.png)
![image](https://user-images.githubusercontent.com/35503088/178497146-9663bdcf-75fe-4d75-8277-3050668b1cde.png)

**Comment by MilosPaunovic** [comment, marked as answer]
```js
@slide="(details) => onSlide(details, customParams)"
```

**Reply by meo1** [reply]
Thanks.
